nnstreamer_example/custom_example_LSTM/dummy_LSTM.c

```diff
@@ -67,9 +68,28 @@ pt_exit (void *private_data, const GstTensorFilterProperties * prop)
 {
   pt_data *data = private_data;
   assert (data);
+
+  g_free (data->info[0].name);
+  g_free (data->info[1].name);
+  g_free (data->info[2].name);
```

Comment:
actually, we don't need these 3 lines for this file, because 'name' is NULL in this case, but these code are no harm.
g_free internally checks NULL.